Gopkg.toml

```diff
@@ -48,6 +48,11 @@
   version = "1.2.0"
   name = "github.com/golang/protobuf"
 
+[[constraint]]
+  branch = "switch-to-gogo"
+  source = "https://github.com/ilackarms/protokit"
+  name = "github.com/pseudomuto/protokit"
```

Comment:
the reason for this change is that `github.com/pseudomuto/protokit` works entirely off of `golang/protobuf`.

however, all of our protos are based off of `gogo/protobuf`, which creates a compile error when trying to use our generated code `solo-kit.pb.go` with `protokit` (which we rely on for convenience functions to parse the proto code gen request).

this switches solo-kit to use a fork of protokit where `golang` is replaced with `gogo`

Reply:
how does versioning work on this fork (i don't see a version constraint here)

Reply:
also, this doesn't seem to be temporary. why not use your fork as the dependency?

Reply:
i'll make it a permanent dependency and update this pr

Reply:
nevermind, update in next pr